specs/airr-schema.yaml

```diff
@@ -1981,3 +2036,34 @@ CloneTree:
         clone_id:
             type: string
             description: Identifier for the clone.
+        newick_string:
+            type: string
+            description: Newick string of the tree
+        node_list:
+            type: array
+            description: Nodes in the tree
+            items:
+                - $ref: '#/Node'
+                
+# 1-to-n relationship between Tree and Node
+Node:
```

Comment:
would want to remove the `$ref`s from `_id` reference fields here as well I think